src/syslog_mcp/services/alert_monitor.py

```
@@ -0,0 +1,357 @@
+        Args:
+            rule: Alert rule configuration
+            
+        Returns:
+            Alert event if triggered, None otherwise
+        """
+        try:
+            # Build Elasticsearch query for the rule
+            time_window_minutes = rule.get('time_window', 60)
+            threshold = rule.get('threshold', 1)
+            query_string = rule.get('query', '')
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)